src/main/java/application/configurations/SecurityConfiguration.java

```diff
@@ -41,14 +41,16 @@ public UserDetailsService userDetailsService(){
      */
     protected void configure(HttpSecurity http) throws Exception {
         http.authorizeRequests()
-                .antMatchers("/survey/create").hasRole("SURVEYOR")
```

Comment:
For the .antMatchers(), I think you want the most restrictive one's (only SURVEYOR) at the top, and have the least restrictive (USER or SURVEYOR) at the bottom. The h2-console thing should be last

Reply:
I thought it only matters when one Ant matcher covers another by using "**"?

When I tried it:
 .antMatchers("/").hasAnyRole("SURVEYOR", "USER")  on first line will get a 403 forbidden when USER role attempt to use /cheat

 .antMatchers("/**").hasAnyRole("SURVEYOR", "USER")  on first line will stop matching all URL after that and allow user to use /cheat, while also require login for /h2-console/

Reply:
Might just be good practice then, even if it doesn't affect the app at this point

Reply:
Order reverted back :)